docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,208 @@
+
+physicalLocation with displayLabel="Address"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/arrow:58/datastream/MODS/view
+
+.. code-block:: xml
+
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+*This is one where I'm not sure where to go with.*
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps "Pi Beta Phi Fraternity" ;
+        opaque:locationShelfLocator "Box 36, Folder 14" .
+
+
+physicalLocation with displayLabel attributes for Collection and Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/arrowmont%3A208/datastream/MODS/view
+
+.. code-block:: xml
+
+
+    <location>
+        <physicalLocation displayLabel="Collection">Archives Collection</physicalLocation>
+        <physicalLocation displayLabel="Repository">Arrowmont School of Arts and Crafts</physicalLocation>
+        <physicalLocation displayLabel="Detailed Location"/>
+        <physicalLocation displayLabel="City">Gatlinburg</physicalLocation>
+        <physicalLocation displayLabel="State">Tennessee</physicalLocation>
+    </location>
+
+*I am not sold on whether retaining the Archives Collection string is necessary. I don't think the city and state are necessary if a URI is used instead of string.*
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2001080757> ;
+        dbo:collection "Archives Collection" .
+
+url with a preview
+^^^^^^^^^^^^^^^^^^
+
+*These only occur in volvoices. Obviously, this is a case where the turtle object URIs will be relative to the new platform.*
+
+https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A9999
+
+.. code-block:: xml
+
+    <location>
+        <url access="object in context" usage="primary display">https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A9999</url>
+        <url access="preview">https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A9999/datastream/TN/view</url>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/objects/1> edm:isShownAt <https://digital.lib.utk.edu/placeholder/shownat/uri> ;
+        edm:preview <https://digital.lib.utk.edu/placeholder/preview/uri> ;
+        edm:object <https://digital.lib.utk.edu/placeholder/object/uri> .
```

Comment:
We had talked about this in our meeting. I'd agree that all of the actual values we have here should be dropped as they will mean nothing in the new system. Mat I think was wondering if more generally we might want to include a field for edm:preview and edm:object as they relate to the new system. I'm not entirely convinced this needs to be in the descriptive metadata (since we won't have the identifiers to create the links until after the objects are initially ingested). I don't think our OAI-PMH process will change radically and I don't see a need to reference the object URL when the user is already viewing the object, but I'm open to talking more about including a place to map these types of values if we think they would be relevant.

Reply:
Here's what I've come around to here:

_The URIs referenced are relative to our current system and would not be migrated. Translating this to RDF would be self-referential and not descriptive metadata. This can be dropped._